lua/entities/gmod_wire_cameracontroller.lua

```diff
@@ -45,16 +45,12 @@ if CLIENT then
 	local enabled = false
 	local self
 
-	local clientprop
-
 	-- Position
 	local pos = Vector(0,0,0)
 	local smoothpos = Vector(0,0,0)
 
 	-- Distance & zooming
 	local distance = 0
-	local curdistance = 0
```

Comment:
This removed local is still referenced. There seems to be some bugs here still. Don't trust the linter blindly.